web/src/portico/integrations_dev_panel.ts

```diff
@@ -7,36 +9,83 @@ import * as channel from "../channel";
 // Data segment: We lazy load the requested fixtures from the backend
 // as and when required and then cache them here.
 
-const loaded_fixtures = new Map();
+const fixture_schema = z.record(
+    z.string(),
+    z.object({
+        body: z.unknown(),
+        headers: z.record(z.string()),
+    }),
+);
+
+type Fixutres = z.infer<typeof fixture_schema>;
+
+type HTMLSelectOneElement = HTMLSelectElement & {type: "select-one"};
```

Comment:
I guess we already have declared this type in `/src/types.ts`?

Reply:
We shouldn't import types/data from web files to a portico file. [Related Comment](https://github.com/zulip/zulip/pull/29947#issuecomment-2093844836)

Reply:
Thanks for the reference, surely this makes then!!!